src/app/Fake.DotNet.Cli/Dotnet.fs

```diff
@@ -0,0 +1,582 @@
+// From https://raw.githubusercontent.com/dolly22/FAKE.Dotnet/master/src/Fake.Dotnet/Dotnet.fs
+// Temporary copied into Fake until the dotnetcore conversation has finished.
+// This probably needs to stay within Fake to bootstrap?
+// Currently last file in FakeLib, until all dependencies are available in dotnetcore.
+/// .NET Core + CLI tools helpers
+module Fake.DotNet.Cli
+
+// NOTE: The #if can be removed once we have a working release with the "new" API
+// Currently we #load this file in build.fsx
+#if NO_DOTNETCORE_BOOTSTRAP
+open Fake.Core
+open Fake.IO.FileSystem
+open Fake.IO.FileSystem.Operators
+#else
+open Fake
+// Workaround until we have a release with the "new" API.
+module Environment =
+    let environVar = environVar
+    let isUnix = isUnix
+module Trace =
+    let trace = trace
+    let traceError = traceError
+    let traceImportant = traceImportant
+    let traceTask s proj = 
+        { new System.IDisposable with 
+            member x.Dispose() = () }
+module String =
+    let replace = replace
+module Process =
+    let ExecProcess = ExecProcess
+    let ExecProcessWithLambdas = ExecProcessWithLambdas
+
+#endif
+open System
+open System.IO
+open System.Security.Cryptography
+open System.Text
+
+/// .NET Core SDK default install directory (set to default localappdata dotnet dir). Update this to redirect all tool commands to different location. 
+let mutable DefaultDotnetCliDir = 
+    if Environment.isUnix
+    then Environment.environVar "HOME" @@ ".dotnet"
+    else Environment.environVar "LocalAppData" @@ "Microsoft" @@ "dotnet"
```

Comment:
that's not ok, location should be resolved from `PATH`, there is no default install